README.md

```diff
@@ -1,24 +1,79 @@
-[gRPC - An RPC library and framework](http://github.com/grpc/grpc)
+gRPC - An RPC library and framework
```

Comment:
nit; I think it would look better if we capitalized the L and F

Reply:
Will leave as is because grpc-java also uses lowercase.
https://github.com/grpc/grpc-java/blob/3a58a9999e33452268d831da8c9ab61f533380b4/README.md